packages/app/src/components/familyInformation/familyInformation.js

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import styles from "./styles.module.css";
+
+function FamilyMemberInfo(props) {
+  const fields = [
+    {
+      label: "Name",
+      value: props.familyRepresentativeName
+    },
+    {
+      label: "Email Address",
+      value: props.familyRepresentativeEmailAddress
+    },
+    {
+      label: "Phone Number",
+      value: props.familyRepresentativePhoneNumber
+    },
+    {
+      label: "Relationship",
+      value: props.representativeRelationship
+    }];
+  return (
+    <div className={styles.main}>
+      <div className={styles.header}>
+        <span className={styles.bold}>Family Member Information</span>
+      </div>
+      <div className={styles.content}>
+        {
+          fields.map((obj) => (<div key={obj.label}>
+            <label>{obj.label}</label>
```

Comment:
i dont think we want this bolded and it is because it is a `label` i think you can just change it to a normal `div`